test/unit/xrow.cc

```diff
@@ -699,6 +741,7 @@ main(void)
 	test_xrow_decode_error_2();
 	test_xrow_decode_error_3();
 	test_xrow_decode_error_4();
+	test_xrow_decode_error_9098();
```

Comment:
Where are tests between 4 and 9098?

Reply:
Renamed to `test_xrow_decode_error_gh_9098`.